inspire_dojson/hep/rules/bd0xx.py

```diff
@@ -66,13 +67,10 @@ def _normalize(medium):
         return normalized_medium
 
     def _get_isbn(value):
-        def _normalize(isbn):
-            return isbn.upper().replace('-', '')
-
-        isbn = force_single_element(value.get('a', ''))
-        normalized_isbn = _normalize(isbn)
-
-        return normalized_isbn
+        a_value = force_single_element(value.get('a', ''))
+        normalized_a_value = a_value.replace('.', '')
+        if normalized_a_value:
+            return str(ISBN(normalized_a_value))
```

Comment:
Done this in both places. I had to use the unsavory `except Exception` because it can raise a variety of errors (see: https://github.com/inspirehep/inspire-schemas/blob/968ad6c36d73c984ce2d2ff6165a1c7654ff4336/inspire_schemas/builders/references.py#L291-L294).